client/pom.xml

```diff
@@ -1,1190 +1,1219 @@
-                <goals>
-                  <goal>heat</goal>
-                </goals>
-                </execution>
-                <execution>
-                  <id>wix-cs-setup-db</id>
-                  <configuration>
-                    <vars>-var wix.SetupPathDir</vars>
-                    <workingDirectory>${basedir}/target</workingDirectory>
-                    <dir>setup</dir>
-                    <template>fragment</template>
-                    <directoryName>CSMANAGEMENT</directoryName>
-                    <outputFile>target/setupdb.wxs</outputFile>
-                    <componentGroup>SetupPath</componentGroup>
-                  </configuration>
-                  <goals>
-                    <goal>heat</goal>
-                  </goals>
-                </execution>
-                <execution>
-                  <id>wix-cs-utilities</id>
-                  <configuration>
-                    <vars>-var wix.UtilitiesPathDir</vars>
-                    <workingDirectory>${basedir}/target</workingDirectory>
-                    <dir>scripts</dir>
-                    <template>fragment</template>
-                    <directoryName>INSTALLDIR</directoryName>
-                    <outputFile>target/utilities.wxs</outputFile>
-                    <componentGroup>UtilitiesPath</componentGroup>
-                  </configuration>
-                  <goals>
-                    <goal>heat</goal>
-                  </goals>
-                </execution>
-                <execution>
-                  <id>python-site-packages</id>
-                  <configuration>
-                    <vars>-var wix.PythonSitePackagesDir</vars>
-                    <workingDirectory>${basedir}/target</workingDirectory>
-                    <dir>python-site-packages</dir>
-                    <template>fragment</template>
-                    <directoryName>INSTALLDIR</directoryName>
-                    <outputFile>target/python-site-packages.wxs</outputFile>
-                    <componentGroup>PythonSitePackagesPath</componentGroup>
-                  </configuration>
-                  <goals>
-                    <goal>heat</goal>
-                  </goals>
-                </execution>
-            </executions>
-          </plugin>
-          <plugin>
-            <groupId>org.bitbucket.joxley</groupId>
-            <artifactId>wix-maven-plugin</artifactId>
-            <version>1.1.1</version>
-            <executions>
-              <execution>
-                <id>wix-acs</id>
-                  <configuration>
-                    <workingDirectory>${basedir}/target</workingDirectory>
-                    <extensions>
-                      <extension>WixFirewallExtension</extension>
-                      <extension>WixUIExtension</extension>
-                      <extension>WixUtilExtension</extension>
-                    </extensions>
-                    <arguments>-dSourceClient=SourceDir\client -dS
```

Comment:
Isn't it going to fail the execution since the arguments are on multiple line now?

Reply:
That's a very good point, I haven't tested `buildw` profile!

Reply:
Isn't this the profile that was intended to build ACs for windows as an executable file?
I never understood this `Wix` project that we have.

Reply:
I believe so. This is actually my first time trying to execute it.

Reply:
Makes sense, its' not even in [here](https://cwiki.apache.org/confluence/display/CLOUDSTACK/How+to+build+CloudStack#HowtobuildCloudStack-BuildingCloudStack). I'll do some digging the history, then I might remove the whole thing.

Reply:
So the wix project and its workflow last updated 4 years ago, and it's fully broken since Embedded Jetty commit (July 2017). although I couldn't manage to make it work after fixing that part of it.
Besides that, I wasn't able to find any guide anywhere that Cloudstack actually can be installed on Windows. ([4.5](http://docs.cloudstack.apache.org/projects/cloudstack-installation/en/4.5/management-server/index.html), [4.6](http://docs.cloudstack.apache.org/projects/cloudstack-installation/en/4.6/management-server/), [4.7](http://docs.cloudstack.apache.org/projects/cloudstack-installation/en/4.7/management-server/index.html), [4.8](http://docs.cloudstack.apache.org/projects/cloudstack-installation/en/4.8/management-server/index.html), [4.9](http://docs.cloudstack.apache.org/projects/cloudstack-installation/en/4.9/management-server/index.html), [4.11](http://docs.cloudstack.apache.org/projects/cloudstack-installation/en/4.11/management-server/index.html))
I'll send an email to ML, but I guess we can safely delete `Wix-*` and `buildw` altogether.

Reply:
That's great, although I think six months is very long for something which is already stalled. Specially that in most cases when retirement proposal happens, that piece of code must have been obsolete for months, if not years.